test/test_ast_generation.py

```diff
@@ -1,129 +1,299 @@
-    ('lambda_pos_only_args_with_default', 'lambda a=0, /: 42\n'),
-    ('lambda_pos_posonly_args', 'lambda a, b, /, c, d: 42\n'),
-    ('lambda_pos_posonly_args_with_default', 'lambda a, b=0, /, c=2: 42\n'),
-    ('lambda_vararg', 'lambda *a: 42\n'),
-    ('lambda_vararg_kwonly_args', 'lambda *a, b: 42\n'),
-    ('list', '[1, 2, a]\n'),
-    ('list_comp', '[i for i in a]\n'),
-    ('list_comp_if', '[i for i in a if b]\n'),
-    ('list_trailing_comma', '[1+2, a, 3+4,]\n'),
-    ('mixed_args', 'def f(a, /, b, *, c):\n    pass\n'),
-    ('mixed_args_with_default', 'def f(a, b=2, /, c=3, *e, f, **g):\n    pass'),
-    ('multiple_assignments', 'x = y = z = 42\n'),
-    ('multiple_assignments_with_yield', 'x = y = z = yield 42\n'),
-    ('multiple_pass', 'pass; pass\npass\n'),
-    ('nonlocal', 'nonlocal a, b\n'),
-    ('pass', 'pass\n'),
-    ('pos_args', 'def f(a, b):\n    pass\n'),
-    ('pos_args_with_default', 'def f(a, b=2):\n    pass\n'),
-    ('pos_only_args', 'def f(a, /):\n    pass\n'),
-    ('pos_only_args_with_default', 'def f(a=0, /):\n    pass\n'),
-    ('pos_posonly_args', 'def f(a, b, /, c, d):\n    pass\n'),
-    ('pos_posonly_args_with_default', 'def f(a, b=0, /, c=2):\n    pass\n'),
-    ('raise', 'raise\n'),
-    ('raise_ellipsis', 'raise ...\n'),
-    ('raise_expr', 'raise a\n'),
-    ('raise_from', 'raise a from b\n'),
-    ('return', 'return\n'),
-    ('return_expr', 'return a\n'),
-    ('set', '{1, 2+4, 3+5}\n'),
-    ('set_comp', '{i for i in a}\n'),
-    ('set_trailing_comma', '{1, 2, 3,}\n'),
-    ('simple_assignment', 'x = 42\n'),
-    ('simple_assignment_with_yield', 'x = yield 42\n'),
-    ('try_except', 'try:\n    pass\nexcept:\n    pass\n'),
-    ('try_except_else', 'try:\n    pass\nexcept:\n    pass\nelse:\n    pass\n'),
-    ('try_except_else_finally', 'try:\n    pass\nexcept:\n    pass\nelse:\n    pass\nfinally:\n    pass\n'),
-    ('try_except_expr', 'try:\n    pass\nexcept a:\n    pass\n'),
-    ('try_except_expr_target', 'try:\n    pass\nexcept a as b:\n    pass\n'),
-    ('try_except_finally', 'try:\n    pass\nexcept:\n    pass\nfinally:\n    pass\n'),
-    ('try_finally', 'try:\n    pass\nfinally:\n    pass\n'),
-    ('tuple', '(1, 2, 3)\n'),
-    ('vararg', 'def f(*a):\n    pass\n'),
-    ('vararg_kwonly_args', 'def f(*a, b):\n    pass\n'),
-    ('while', 'while a:\n    pass\n'),
-    ('while_else', 'while a:\n    pass\nelse:\n    pass\n'),
-    ('with', 'with a:\n    pass\n'),
-    ('with_as', 'with a as b:\n    pass\n'),
-    ('yield', 'yield\n'),
-    ('yield_expr', 'yield a\n'),
-    ('yield_from', 'yield from a\n'),
+    ('annotated_assignment', 'x: int = 42'),
+    ('annotated_assignment_with_parens', '(paren): int = 3+2'),
+    ('annotated_assignment_with_yield', 'x: int = yield 42'),
+    ('annotated_no_assignment', 'x: int'),
+    ('annotation_with_parens', '(parens): int'),
+    ('assert', 'assert a'),
+    ('assert_message', 'assert a, b'),
+    ('asyncfor', '''\
```

Comment:
Can you do something about the ugly `\` too here?

Reply:
Done in 81c5b34